authn_k8s_scale_demo/build/copy-summon.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
+cp /usr/local/bin/summon /summon/
```

Comment:
Why not just `mv` Summon into `/summon/` in the `Dockerfile` instead of first untarring it into `/usr/local/bin/summon`? Does it need to exist in both places?

Reply:
The directory `/summon` is a memory volume which is shared with the application container. By copying it here, the authenticator container can provide Summon to the application container so that the application container doesn't have to have Summon built into it.